helm/sematic-server/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sematic-server
 description: Sematic AI Server
 type: application
-version: 1.0.3
+version: 1.0.4
```

Comment:
Might need to revert this before merge

Reply:
We probably need a development naming scheme.
cc @snoshy